Zpp/Test/Integration_Tests/TestPerformance.cs

```diff
@@ -8,7 +9,7 @@ public class TestPerformance : AbstractTest
         
         private const int MAX_TIME_FOR_MRP_RUN = 90;
 
-        public TestPerformance()
+        public TestPerformance() : base(true, true)
```

Comment:
This is dangerous, can be easily swapped. I would prefer to configure usage of local db globally e.g. in Configuration class in Zpp\Configuration.

Reply:
made it more Readable, but it should be set with Environment Variable because it is just for my local setup.

Reply:
Changed it to Environment Variable.